hq/app/aws/ec2/EC2.scala

```diff
@@ -67,6 +70,13 @@ object EC2 {
     } yield sortSecurityGroupsByInUse(flaggedSgsWithVpc, sgUsage)
   }
 
+  def refreshSGSReports(accounts: List[AwsAccount])(implicit ec: ExecutionContext): Attempt[List[Either[FailedAttempt, RefreshTrustedAdvisorCheckResult]]] = {
```

Comment:
This returns all the failures and successes, but we're only throwing the result away. Should we be doing something with any failures we get here and if not, could we have this function return something simpler?